packaging/installer/methods/kubernetes.md

```diff
@@ -0,0 +1,134 @@
+The Helm chart installs one parent pod for storing metrics and managing alarm notifications plus an additional child pod
+for every node in the cluster.
+
+Each child pod will collect metrics from the node it runs on in addition to [22 supported
+services](https://github.com/netdata/helmchart#service-discovery-and-supported-services) via [service
+discovery](https://github.com/netdata/agent-service-discovery/). Each child pod will also collect
+[cgroups](/collectors/cgroups.plugin/README.md),
+[Kubelet](https://learn.netdata.cloud/docs/agent/collectors/go.d.plugin/modules/k8s_kubelet), and
+[kube-proxy](https://learn.netdata.cloud/docs/agent/collectors/go.d.plugin/modules/k8s_kubeproxy) metrics from its node.
+
+To install Netdata on a Kubernetes cluster, you need:
+
+-   A working cluster running Kubernetes v1.9 or newer.
+-   The [Helm package manager](https://helm.sh/) on an administrative system.
+-   The [kubectl](https://kubernetes.io/docs/reference/kubectl/overview/) command line tool on the same system.
+
+The default configuration creates four pods in your cluster: three `child` pods and one `parent` pod. The `child` pods
+collect metrics and stream the information to the `parent` pod, which uses two persistent volumes to store metrics and
+alarms. The `parent` pod also handles alarm notifications and enables the Netdata dashboard using an ingress controller.
+
+## Install the Netdata Helm chart
+
+Download the [Netdata Helm chart](https://github.com/netdata/helmchart) on the administative system where you have the
+`helm` binary installed.
+
+```bash
+git clone https://github.com/netdata/helmchart.git netdata-helmchart
+```
+
+> You may not need to configure the Helm chart to get a functioning service on your cluster, but you should read the
+> sections on [configuring the Helm chart](#configure-the-netdata-helm-chart) and [configuring service
+> discovery](#configure-service-discovery) for details.
+
+Install the Helm chart to your cluster with `helm install`, replacing `--name netdata` with the release name of your
+choosing:
+
+```bash
+helm install --name netdata ./netdata-helmchart
+```
+
+Run `kubectl get services` and `kubectl get pods` to confirm that your cluster now runs a `netdata` service, one
+`parent` pod, and three `child` pods.
+
+You've now installed Netdata on your Kubernetes cluster. See how to [access the Netdata
+dashboard](#access-the-netdata-dashboard) to confirm it's working as expected, or see the next section to [configure the
+Helm chart](#configure-the-netdata-helm-chart) to suit your cluster's particular setup.
+
+## Configure the Netdata Helm chart
+
+Read up on the various configuration options in the [Helm chart
+documentation](https://github.com/netdata/helmchart#configuration) to see if you need to change any of the options based
+on your cluster's setup.
+
+To change a setting, you can edit the `values.yml` file inside of the `netdata-helmchart` folder. Then install the chart
```

Comment:
Preferred way to set a custom setting for a chart is to pass those settings as `--set` args or `--values` arg to helm install:

```
helm install --set a.b.c=xyz netdata netdata/netdata
```

Reply:
What's the recommended way to edit the `sdconfig/slave.yml` file once that's enabled in the Helm chart? Are those values changeable via the same `--set ...` command?

Reply:
@ilyam8 Do you have any insights here? To change the service discovery config, do you edit `sdconfig/child.yml`, or do you use the `--set` command?

Reply:
> What's the recommended way to edit the `sdconfig/slave.yml` file once that's enabled in the Helm chart? Are those values changeable via the same `--set ...` command?

One way we do it for cloud currently, is to use `--values sdconfig/slave.yml` and then override any other config further with `--set`.

Reply:
> What's the recommended way to edit the `sdconfig/slave.yml` file once that's enabled in the Helm chart? Are those values changeable via the same `--set ...` command?

For this, the user can upload an updated `sdconfig/slave.yml` containing their changes, to a new configmap and reference that configmap via `sd.child.configmap.name` and `sd.child.configmap.key` configs.

Reply:
I dont think it is possible to use `--set`, however there is `--set-file`

I created a PR

> https://github.com/netdata/helmchart/pull/109

When it is merged it will be possible to use `--set-file`


```
... --set-file sd.child.configmap.from.value=../child.yml
```

Reply:
i think it is not correct to edit `/sdconfig/child.yml`, you use it as a reference, copy, edit and `--set-file` the new file.

Reply:
Okay, I added some additional steps around editing a copy of `child.yml` and use `--set-file`, and we can merge this once netdata/helmchart#109 is merged and we know the process works as expected.